acct-creation-script/onboard.py

```diff
@@ -226,13 +230,13 @@ def get_assumed_session(environment, role):
     region_name = environment[1]
     boto_session = Session(region_name=region_name)
     sts = boto_session.client('sts', region_name=region_name) #If using opt-in regions, must have AWS_STS_REGIONAL_ENDPOINTS= 'regional' set
-    print(f'ID: {sts.get_caller_identity()}')
-    role_arn = f'arn:aws:iam::{environment[0]}:role/{role}'
-    assumed = sts.assume_role(RoleArn=role_arn, RoleSessionName='arpio_provisioning')
+    aws_account = environment[0]
+    role_arn = f'arn:aws:iam::{aws_account}:role/{role}'
+    assumed = dict(sts.assume_role(RoleArn=role_arn, RoleSessionName='arpio_provisioning'))
```

Comment:
Using a `dict()` here creates a new dictionary, initializing it with the items (keys and values) in the argument (which is probably a `dict` returned by `sts.assume_role()`).  That doesn't seem necessary here.